src/main/resources/page/js/dummy-page-loader.js

```diff
@@ -0,0 +1,11 @@
+class DummyPageLoader {
+    constructor() {}
+
+    static async pageLoader() {
```

Comment:
As this is a method, the name of the method should/must/need (to) start with a verb. ("load" name can be enough)